modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoverySpiMBean.java

```diff
@@ -94,11 +94,11 @@
     public int getThreadPriority();
 
     /**
-     * Gets IP finder clean frequency.
+     * Gets frequency which coordinator clean IP finder and keeps it in the correct state, unregistering addresses of the nodes that has left the topology.
      *
      * @return IP finder clean frequency.
      */
-    @MXBeanDescription("IP finder clean frequency.")
+    @MXBeanDescription("Gets frequency which coordinator clean IP finder and keeps it in the correct state, unregistering addresses of the nodes that has left the topology.")
```

Comment:
_The_ frequency _with_ which coordinator clean_s_ IP finder and keeps it in the correct state, unregistering addresses of the nodes that _have_ left the topology.